lib/meadow/data/schemas/validations.ex

```diff
@@ -8,22 +8,27 @@ defmodule Meadow.Data.Schemas.Validations do
   If a `cast_embed()` will result in a `nil` value (either on create or
   update), set it to an empty embedded struct instead
   """
-  def prepare_embed(%Ecto.Changeset{data: data, params: params} = change, field) when is_atom(field) do
+  def prepare_embed(%Ecto.Changeset{data: data, params: params} = change, field)
```

Comment:
Can we please write a test for this? I'm not clear on what is going on here.

Reply:
Yep. I'll see what I can do.